connect-example/src/main/res/values-night/colors.xml

```diff
@@ -1,59 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="default_border_color">#FBD992</color>
```

Comment:
👀 The deleted colors weren't actually different from the base values, and therefore redundant.